lms/static/styles/components/_StudentSelector.scss

```diff
@@ -6,11 +6,13 @@
   display: flex;
 }
 
-.StudentSelector-change-student-svg {
+.StudentSelector__change-student-svg {
   color: var.$grey-5;
+  width: 24px;
+  height: 24px;
```

Comment:
the left/right arrow icons pulled in from the shared package are 16x16, rather than 24x24, this forces them to be 24x24

Ideally we should use a mixin utility such as 

```@mixin icon--medium {
  width: var.$icon-size--medium;
  height: var.$icon-size--medium;
}
```

But these only exist on the client at present. Its likely we may bring them into the shared package at some point, but I think hard-coding the values for now is okay.

Reply:
As we move forward, icon (SVG) sizing should be handled with proportional (`em`) units, based on the contextual local `font-size`. Thus, all icons should be sized `1em` both in width and height, and the pixel size of that would be based on the `font-size`. You can see this is the case in the styling for the shared button components, for example. 

What I'd suggest for now (until we can be more systematic here) is:

* Set a `font-size` on an appropriate parent element. You can hard-code `24px` for now if there is no better way to do it. I'll leave that up to you.
* Set width/height rules on this selector to `1em`

The longer-term fix will involve updating how `SvgIcon` styling works. But we'll need an interim step.

The main goal being: SVG sizing based on `em` units based on local font context.